crates/consensus/src/engine/mod.rs

```diff
@@ -395,7 +395,7 @@ mod tests {
             });
             assert_matches!(
                 engine.try_construct_block(block_with_zero_base_fee.into()),
-                Err(EngineApiError::PayloadBaseFee(val)) if val == 0.into()
+                Err(EngineApiError::PayloadBaseFee(val)) if val == U256::ZERO
```

Comment:
is there an `is_zero()` function?

if not, imo worth adding this to ruint

Reply:
yeah agreed let's add it